exporters/elasticsearch-exporter/src/main/java/io/zeebe/exporter/ElasticsearchClient.java

```diff
@@ -118,7 +125,10 @@ public void bulk(final Map<String, Object> command, final Record<?> record) {
           "Failed to serialize bulk request command to JSON", e);
     }
 
-    bulkRequest.add(serializedCommand + "\n" + record.toJson());
+    final String jsonCommand = serializedCommand + "\n" + record.toJson();
+    if (bulkRequest.isEmpty() || !bulkRequest.get(bulkRequest.size() - 1).equals(jsonCommand)) {
```

Comment:
We could add a comment here why we're filtering it out. I think it is not so obvious.